### PR TITLE
Fix wrong file check from grunt

### PR DESCRIPTION
**Pull request by sunglim**
More fix : https://github.com/beagleterm/beagle-term/issues/59

@RandomlyKnighted

### CODE REVIEW

File: app/Gruntfile.js [hunk 1]
```diff
@@ -5,9 +5,7 @@ module.exports = function(grunt) {
     pkg: '<json:package.json>',
     jshint: {
       all: {
-        src: [
-          'js/index.js', 'js/main.js'
-        ]
+        src: ['js/*.js']
```
Comment (RandomlyKnighted) [inline app/Gruntfile.js]: This won't test hterm will it?

Comment (sunglim) [inline app/Gruntfile.js]: This doesn't test hterm.

Here's the result. only two file is tested.  Do you think we have to specify each file?

$ grunt test
Running "jshint:all" (jshint) task
$$ 2 files lint free

Comment (RandomlyKnighted) [inline app/Gruntfile.js]: Nope looks good. I just couldn't remember if using wildcards like that made it check subfolders or not. Looks like it doesn't which is what we want.

**Comment by sunglim** [comment]
Thanks for the review!

**Comment by RandomlyKnighted** [comment]
Happy to help.

**Pull request merged**